### PR TITLE
merge chronic inflammatory demyelinating polyneuritis

### LINKED ISSUE #7119: MONDO:0002335 & MONDO:0006702 [Merge]
**Mondo term (ID and Label)**
MONDO:0002335 chronic inflammatory demyelinating polyneuritis

**Reason for deprecation**
SNOMEDCT [:128209004](http://snomed.info/id/128209004)
MeSH [D020277](http://id.nlm.nih.gov/mesh/D020277)
and Orphanet [Orphanet:2932](http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=2932)
and NCI C84636 http://purl.obolibrary.org/obo/NCIT_C84636
suggest these may be synonyms.

**Term to be merged with**
chronic inflammatory demyelinating polyradiculoneuropathy MONDO:0006702

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7119

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -125013,31 +124996,41 @@ id: MONDO:0006702
 name: chronic inflammatory demyelinating polyradiculoneuropathy
 def: "A rare neurological disorder in which there is inflammation of nerve roots and peripheral nerves and destruction of the fatty protective covering (myelin sheath) over the nerves. This affects how fast the nerve signals are transmitted and leads to loss of nerve fibers. This causes weakness, paralysis and/or impairment in motor function, especially of the arms and legs (limbs). Sensory disturbance may also be present. The motor and sensory impairments usually affect both sides of the body (symmetrical), and the degree of severity and the course of disease may vary from case to case. Some affected individuals may follow a slow steady pattern of symptoms while others may have symptoms that stabilize and then relapse." [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/]
 subset: gard_rare {source="GARD:6102"}
+subset: inferred_rare
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:124999]: @nicolevasilevsky should this `subset` be added? In a previous [ "split" PR](https://github.com/monarch-initiative/mondo/pull/7320/files#r1501269691) Sabrina said subset tags should not be added.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -125013,31 +124996,41 @@ id: MONDO:0006702
...
 subset: orphanet_rare {source="Orphanet:2932"}
 subset: rare
+synonym: "chronic inflammatory demyelinating polyneuritis" EXACT [DOID:2536]
+synonym: "chronic inflammatory demyelinating polyneuropathy" EXACT [NCIT:C84636]
 synonym: "chronic inflammatory demyelinating polyradiculoneuropathy" EXACT [Orphanet:2932]
 synonym: "chronic relapsing polyneuropathy" EXACT [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/]
 synonym: "CIDP" EXACT ABBREVIATION [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/, OMIM:139393, Orphanet:2932]
+xref: DOID:2536
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:125007]: missing `source`

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -125013,31 +124996,41 @@ id: MONDO:0006702
...
 xref: GARD:6102 {source="Orphanet:2932"}
-xref: ICD10CM:G61.81 {source="DOID:5213", source="MONDO:directSiblingOf"}
-xref: ICD9:357.81 {source="DOID:5213", source="MONDO:directSiblingOf"}
+xref: ICD10CM:G61.81 {source="DOID:5213", source="MONDO:equivalentTo", source="DOID:2536"}
+xref: ICD9:357.81 {source="DOID:5213", source="MONDO:equivalentTo", source="MONDO:i2s", source="DOID:2536"}
 xref: ICD9:357.89 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MedDRA:10057645 {source="EFO:1000868"}
 xref: MESH:D020277 {source="DOID:5213", source="MONDO:equivalentTo", source="EFO:1000868"}
-xref: NCIT:C84636 {source="DOID:5213", source="MONDO:directSiblingOf"}
+xref: MESH:D020277 {source="DOID:2536"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:125016]: should this line `xref: MESH:D020277 {source="DOID:2536"}` have been combined with `xref: MESH:D020277 {source="DOID:5213", source="MONDO:equivalentTo", source="EFO:1000868"}`?

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -125013,31 +124996,41 @@ id: MONDO:0006702
...
+xref: NCIT:C84636 {source="DOID:5213", source="MONDO:equivalentTo", source="DOID:2536"}
 xref: Orphanet:2932 {source="MONDO:equivalentTo"}
-xref: SCTID:128209004 {source="DOID:5213", source="MONDO:equivalentTo", source="EFO:1000868"}
-xref: SCTID:444728005 {source="DOID:5213", source="MONDO:directSiblingOf"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:125032]: why was the xref to ` SCTID:444728005` removed from both terms?

**Review by twhetzel — CHANGES_REQUESTED** [review]
i think a few things are missing

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:124999]: @sabrinatoro these rare tags came from the obsoleted/merged term- should I remove them or it is okay to move them from the obsolete term to the merged term in cases of a merge?

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:125007]: I will add a MONDO:relatedTo tag

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:125016]: yes, good catch!

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:125032]: it is obsolete (no results found when I searched SNOMED)

**Comment by nicolevasilevsky** [comment]
@twhetzel this is ready for re-review

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:124999]: @nicolevasilevsky did you hear back from Sabrina or do you want to merge this one as-is for now and we'll sort out the policy on subset at next week's curation call?

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:124999]: I haven't heard back, let's merge before we get conflicts and we can revise later if needed. Good idea, thanks!

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:124999]: Sounds good!

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**